Directory.Build.props

```diff
@@ -7,10 +7,8 @@
     -->
 
   <PropertyGroup>
-    <ReactNativeWindowsDir>$(MSBuildThisFileDirectory)\vnext\</ReactNativeWindowsDir>
-    <ReactNativePackageDir>$(MSBuildThisFileDirectory)\node_modules\react-native\</ReactNativePackageDir>
```

Comment:
just to confirm, do we not need this ReactNativePackageDir because it's defined in vnext's directory.build.props?

Reply:
The root Directory,Build.Props is only included when building inside of our repo. It is not included in the `react-native-windows` package that customers build. That is only the one in `vnext`.

Reply:
The comment that was in here was misleading, but there is a reason why we need part of it. Adding back...